### PR TITLE
fix: reset Measurement tool on unmount

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**
#### Type of change

Bug 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/UX-1290

## Description :pencil:

Fix bug where starting a measurement, and then navigating out of viewer mode, causing toolbar to unmount, would not cancel measurement properly.

## How has this been tested? :mag:

In Toolbar story. 

## Test instructions :information_source:

Ensure clicking the button a second time cancels measurements.
Ensure navigating to other story after starting measurement, cancels without errors.

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/components/RevealToolbar/MeasurementButton.tsx [hunk 1]
```diff
@@ -39,25 +39,19 @@ export const MeasurementButton = (): ReactElement => {
...
+  const handleMeasurement = (_enable: boolean): void => {
     if (viewer.models.length <= 0) {
       return;
     }
     setMeasurementEnabled((prevState) => !prevState);
-    if (enable) {
-      enterMeasurement();
-    } else {
-      exitMeasurement();
-    }
```
Comment (Savokr) [inline react-components/src/components/RevealToolbar/MeasurementButton.tsx:51]: Why we don't need that part anymore?

**Review by Savokr — APPROVED** [review]
Looks good!

File: react-components/src/components/RevealToolbar/MeasurementButton.tsx (hunk 1 above)
Comment (haakonflatval-cognite) [reply react-components/src/components/RevealToolbar/MeasurementButton.tsx:51]: It's automatically handled by the `useEffect`

**Review by Savokr — APPROVED** [review]
👍

**Pull request merged**